randovania/data/migration_data.json

```diff
@@ -2406,6 +2406,204 @@
             }
         }
     ],
+    "cave_story": [
```

Comment:
Since there's not released version with old cave story presets/rdvgames, we don't really these. (The same applies to the Dread/SM data tbh)

Reply:
I needed this during dev, I can take it out at the end if you insist